internal/generator.go

```diff
@@ -0,0 +1,68 @@
+package internal
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/protobuf/compiler/protogen"
+	"google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/types/pluginpb"
+)
+
+const version = "1.0.0"
```

Comment:
Would be nice if we can somehow override this with the current git tag / commit hash,

Reply:
Never done this before, but I suppose we could use [runtime/debug.BuildInfo](https://pkg.go.dev/runtime/debug#BuildInfo) to do this. Let me see if I can change this.